### PR TITLE
Mathematical Analysis complete

### PR DESCRIPTION
**Pull request by Joseph-Burley**
The mathematical analysis section goes in depth on how to find the constants for the harmonics. The other section is a paragraph about how the labview file was set up and how we got the graphs. Feel free to make changes or suggestions.

### CODE REVIEW

**Comment by theNerd247** [comment]
Great work! I like the math derivations!

File: lab2/mathematicalAnalysis.tex [hunk 1]
```diff
@@ -28,5 +28,29 @@
...
+	$$cos((2k-1) \pi) = -1 \text{ for all $k$}$$
+	$$b_k = \frac{-2}{(2k-1) \pi} \Big(-2 \Big)$$
+	$$b_k = \frac{4}{(2k-1) \pi}$$
+	\\
+	Substituting $\omega_0 = \frac{2 \pi}{T}$ and $f = \frac{1}{T}$ into the sine terms of the earlier definition of the Fourier Series we get,
+	
+	$$f(t) = \sum_{k=1}^{\infty} \frac{4}{(2k-1) \pi} sin\Big(\frac{2 \pi (2k-1)}{T}t \Big)$$
+	
+	
+	For $k=1$   $1.2732 sin(2 \pi f)$
```
Comment (theNerd247) [inline lab2/mathematicalAnalysis.tex]: What was the intention of this paragraph? Also was this supposed to be a table? If not the formatting is a bit off on my machine (the spacing between the constants and the sines are really crammed together.

**Comment by Joseph-Burley** [comment]
The purpose of that paragraph was to show explicitly how and why I went from using n as the index to k. The portion where it says 
k=1 ...
k=2 ...
etc. 
is supposed to be a table, but I was writing that at 8:30 and didn't want to look up how to do a table.

**Comment by theNerd247** [comment]
Ok. I fixed it

**Pull request closed without merging**